src/views/count.vue

```diff
@@ -100,6 +100,10 @@
         product: "product/getCurrent"
       })
     },
+    mounted(){
+      this.fetchProduct(this.$route.params.sku)
+      console.log("mounted", this.$route)
```

Comment:
Remove console log statement

Reply:
Removed Sir,
Thank you sir for reviewing my code.